### ISSUE TITLE
Snippet in custom command doesn't work with function metadata

### ISSUE THREAD

**Issue by sashton** [BetterThanTomorrow/calva#1463]
`$top-level-defined-symbol` does not return the function symbol when there is metadata declared on the function.

Sample custom command definition:
```
    {
      "name": "print function name",
      "key": "printFunc",
      "snippet": "(println '$top-level-defined-symbol)"
    },
```

# Failing Scenario (when metadata is present)

```
(defn ^{:foo "bar"} baz
  []
  true)
```

Ouput for custom command when cursor is in `baz` function body:
```
(println ')
; Syntax error reading source at (REPL:1293:12).
; Unmatched delimiter: )
```

# Functional Scenario (when no metadata is present)
```
(defn baz
  []
  true)
```

Ouput for custom command when cursor is in `baz` function body:
```
(println 'baz)
baz
nil
```

Calva version: v2.0.233

**Comment by sashton** [comment]
I'm working on a PR for this.

**Comment by PEZ** [comment]
I think this was fixed by #1466
